app/models/manageiq/providers/azure/cloud_manager/orchestration_template.rb

```diff
@@ -1,4 +1,8 @@
 class ManageIQ::Providers::Azure::CloudManager::OrchestrationTemplate < ::OrchestrationTemplate
+  def format
+    'json'.freeze
+  end
```

Comment:
What's this for?

Reply:
It is for near future use. All template classes will have this method to tell the format of the content. For Azure only json is supported.

Reply:
ah this will pave the way for yaml format soon Im guessing.